bitcoin/src/blockdata/transaction.rs

```diff
@@ -1053,6 +1053,10 @@ impl InputWeightPrediction {
     }
 
     /// Computes the prediction for a single input.
+    ///
+    /// # Returns
+    ///
+    /// `InputWeightPrediction` with `input_script_len` and `witness_element_lengths` calculated.
```

Comment:
In a7d8e4df34a9664a834cd71d8dfba83509ee241a:

I don't understand this new comment. The two fields you mention are inputs to the function, not return values.

Reply:
Ok I moved the documentation update to the functions where it is more appropriate https://github.com/rust-bitcoin/rust-bitcoin/pull/4626/commits/f1f8245e44a2a755c5a24a2d16d8b43ded8654cf

Reply:
I think this will still _technically_ panic if `usize` is <= `u32`, so I could see an argument for adding that to the docs someplace as well.  Doesn't need to be done with this PR though.

Reply:
marking resolved